Containerfile.rag-content

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/ubi9/python-311 as lightspeed-rag-builder
+
+USER 0
+RUN dnf install -y --nodocs --setopt=keepcache=0 --setopt=tsflags=nodocs git ruby gem
+RUN pip install torch --index-url https://download.pytorch.org/whl/cpu
+RUN pip install PyYAML huggingface_hub llama_index langchain_community \
+                faiss-cpu llama-index-vector-stores-faiss sentence_transformers \
+                llama-index-embeddings-huggingface
+RUN gem install asciidoctor
+
+WORKDIR /workdir
+RUN git clone --single-branch --branch enterprise-4.14 https://github.com/openshift/openshift-docs.git
+COPY scripts/asciidoctor-text ./asciidoctor-text
+RUN python asciidoctor-text/convert-it-all.py \
+           -i openshift-docs \
+           -t openshift-docs/_topic_maps/_topic_map.yml \
+           -o ocp-product-docs-plaintext \
+           -a asciidoctor-text/attributes.yaml
```

Comment:
@thoraxe you've been more involved than me in getting docs content we can generate embeddings from....any of this surprising to you?